docs/docs/how-to/styling/tailwind-css.md

```diff
@@ -2,214 +2,29 @@
-
-```css:title=src/css/index.css
-@tailwind base;
-@tailwind components;
-@tailwind utilities;
-```
-
-**Note**: if using SCSS (or another supported language, rename files/folders appropriately).
-
-2. Import any custom CSS files or add any custom CSS you need (optional)
-
-```css:title=src/css/index.css
-@tailwind base;
-@tailwind components;
-@tailwind utilities;
-
-@import popup.css body {
-  @apply bg-purple-200;
-}
-```
-
-Tailwind will swap these directives out at build-time with all of the styles it generates based on your configured design system.
-
-3. Import this file into your `gatsby-browser.js`
-
-In `gatsby-browser.js` add an import rule for your Tailwind directives and custom CSS to pull them into your site.
-
-```js:title=gatsby-browser.js
-import "./src/css/index.css"
-```
-
-### 4. Configuring your content path
-
-By default, Tailwind ensures that only the classes we need are delivered to the browser. Rather than including every combination of every class you might think of, Tailwind automatically removes unused classes. Because of this, it requires a configuration file to tell it which content to scan.
-
-**3.0.0 and above**
-
-You can mark files to process directly from your Tailwind config. You need to provide an array of strings telling it which files to process.
-
-```js:title=tailwind.config.js
-module.exports = {
-  content: ["./src/**/*.{js,jsx,ts,tsx}"],
-  theme: {},
-  variants: {},
-  plugins: [],
-}
-```
-
-See Tailwind's documentation for [usage with Gatsby](https://tailwindcss.com/docs/guides/gatsby) and [extended content configuration options](https://tailwindcss.com/docs/content-configuration) for more information.
-
-> **Note:** It is **not recommended** that you include Gatsby's output directories (e.g. `public`, `.cache`) in your content array in your Tailwind config. You should only need to include your source files to have Tailwind work as expected.
-
-**Older versions**
-
-It is recommended you install the latest version of Tailwind CSS to get all available features. If you need to use an older version, you can follow the instructions on the PurgeCSS website - [Purge CSS manually in older Tailwind versions](https://purgecss.com/plugins/gatsby.html#installation)
+Please follow the [official "Install Tailwind CSS with Gatsby" guide](https://tailwindcss.com/docs/guides/gatsby) to install and configure Tailwind CSS with Gatsby. Some important notes when configuring Tailwind with Gatsby:
+
+- It is **not recommended** that you include Gatsby's output directories (`public` and `.cache`) in your `content` array in your `tailwind.config.js`. You should only include your source files to have Tailwind working as expected.
+- If you use [GraphQL Typegen](/docs/how-to/local-development/graphql-typegen/) a file at `src/gatsby-types.d.ts` will be generated and with the default configuration for `content` in `tailwind.config.js` this will trigger an infinite loop. You have two options to fix this:
```

Comment:
Does "default configuration" here mean `content` is an empty array (what it is after executing `npx tailwindcss init -p`) or an array with `"./src/**/*.{js,jsx,ts,tsx}"`? If the latter can we add a diff to the snippet in the second list item below

Reply:
The latter. I'm referring to the Tailwind CSS guide. `diff` is a good idea!